app/controllers/companies_controller.rb

```diff
@@ -27,6 +22,12 @@ def set_companies
   end
 
   def set_articles
-    company.articles.published.sorted.page params[:page]
+    articles = company.articles.published
```

Comment:
```ruby
def sorted_articles
  filtered_articles.sorted
end

def filtered_articles
  FilteredArticlesQuery.new(articles, filter_params).all
end

def raw_relation
  company.articles.published.page(params[:page]).per(10)
end
```

Reply:
also please rename `.sorted` method to more specific name like `.sorted_by_name`